doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1114 @@
+    To disable it, set both :kconfig:option:`CONFIG_SECURE_BOOT` and :kconfig:option:`CONFIG_BUILD_S1_VARIANT` Kconfig options to ``n``.
+  * The :ref:`lib_nrf_cloud` library can now handle modem FOTA updates if :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` is enabled.
+
+nRF9160: Serial LTE modem
+-------------------------
+
+* Added:
+
+  * ``#XDFUGET`` and ``#XDFURUN`` AT commands to support the cloud-to-nRF52 DFU service.
+  * Native TLS support to the HTTPS client.
+  * ``#XCMNG`` command to support the use of native TLS.
+  * ``#XSOCKETSELECT`` AT command to support multiple sockets in the Socket service.
+  * ``#XPOLL`` AT command to poll selected or all sockets for incoming data.
+
+* Updated:
+
+  * Enhanced the ``#XHTTPCREQ`` AT command for better HTTP upload and download support.
+  * Enhanced the ``#XSLEEP`` AT command to support data indication when idle.
+  * Enhanced the MQTT client to support the reception of large PUBLISH payloads.
+  * The :ref:`lib_nrf_cloud` library is now used directly instead of the Cloud API.
+
+* Fixed:
+
+  * The secondary MCUboot partition information is no longer passed to the P-GPS library if the P-GPS partition is enabled.
+  * The combined use of A-GPS and P-GPS so that ephemeris and almanac data is not requested through A-GPS, saving both power and bandwidth.
+
+nRF Machine Learning (Edge Impulse)
+-----------------------------------
+
+* Increased the value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds for Thingy:53.
+
+nRF Desktop
+-----------
+
+* Added managing BLE connection interval depending on the USB state to reduce power consumption when USB is suspended.
+* Changed default Bluetooth connection interval from 7.5 ms to 10 ms for nRF Desktop centrals that support LLPM and two simultaneous Bluetooth connections.
+  This is needed to prevent Bluetooth Link Layer scheduling issues.
+* Removed configurations without a bootloader.
+  The B0 bootloader is enabled by default on all boards if the configuration with two image slots fits in memory.
+  Alternatively, MCUboot bootloader with a single image slot and serial recovery is enabled.
+  In case the configuration with the MCUboot does not fit in memory, no bootloader is enabled.
+
+Thingy:53 Zigbee weather station
+--------------------------------
+
+* Added new :ref:`zigbee_weather_station_app` application.
+* Fixed an issue where the buffer was being freed incorrectly.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <sample>`, including protocol-related samples.
+For lists of protocol-specific changes, see `Protocols`_.
+
+Bluetooth samples
+-----------------
+
+* Added:
+
+  * :ref:`peripheral_ams_client` sample.
+  * :ref:`peripheral_fast_pair` sample.
+    See :ref:`ug_bt_fast_pair` for details about this feature.
+
+* Removed Peripheral Alexa Gadgets Bluetooth sample due to Amazon pausing support for the Gadgets ecosystem.
+
+* Updated:
+
+  * :ref:`direct_test_mode` sample:
```

Comment:
```suggestion
* :ref:`direct_test_mode` sample:
```
I thought we agreed to remove the Updated bullet point for lines where bullet points are pointing to specific sample or library. Global edit.

Reply:
done globally